types.d.ts

```diff
@@ -0,0 +1,17 @@
+import type { Compiler, WebpackPluginInstance } from "webpack";
+
+interface Options {
+  /**
+   * If true then generate a top level function declaration statement with comment.
+   */
+  comment: boolean;
+  /**
+   * Array of source file paths that to generate global assignments expression from exports.* statements.
+   */
+  autoGlobalExportsFiles: string[];
```

Comment:
Can this be changed to optionally property?
```suggestion
  autoGlobalExportsFiles?: string[];
```